ui/components/app/shield-entry-modal/shield-entry-modal.tsx

```diff
@@ -86,18 +100,35 @@ const ShieldEntryModal = ({
   };
 
   const handleOnGetStarted = () => {
-    handleOnClose('get-started');
+    handleOnClose(ShieldCtaActionClickedEnum.Start14DayTrial);
+
+    captureShieldCtaClickedEvent({
+      source: ShieldCtaSourceEnum.Homepage, // FIXME: get the correct source
+      ctaActionClicked: ShieldCtaActionClickedEnum.Start14DayTrial,
+      redirectToPage: SHIELD_PLAN_ROUTE,
+    });
+
     navigate(SHIELD_PLAN_ROUTE);
   };
 
+  const handleOnLearnMoreClick = () => {
+    captureShieldCtaClickedEvent({
```

Comment:
here, the modal does close because of learn more click in some cases
So, pls add an event for unmount